flytekit/core/promise.py

```diff
@@ -965,9 +965,8 @@ def create_and_link_node_from_remote(
     for k in sorted(typed_interface.inputs):
         var = typed_interface.inputs[k]
         if k not in kwargs:
-            if _inputs_not_allowed and _ignorable_inputs:
```

Comment:
if `_inputs_not_allowed` is None, which means no fixed inputs the exception is thrown, so is not possible to use a remote LP without fixed inputs even if all inputs are provided using `*kwargs` and default inputs.
https://github.com/flyteorg/flytekit/blob/b85a4fca36ccc47c5bd9ca1ef74043af1e5eae2d/flytekit/core/promise.py#L971